applications/FluidDynamicsApplication/python_scripts/python_solvers_wrapper_fluid.py

```diff
@@ -8,7 +8,8 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
     solver_type = solver_settings["solver_type"].GetString()
 
     if solver_type == "ale_fluid":
-        import navier_stokes_ale_fluid_solver
+        # This include NEEDS to be here bcs of its dependencies
+        import KratosMultiphysics.FluidDynamicsApplication.navier_stokes_ale_fluid_solver
```

Comment:
```suggestion
        from KratosMultiphysics.FluidDynamicsApplication import navier_stokes_ale_fluid_solver
```

otherwise won't work